packages/amplify-e2e-tests/src/__tests__/api_3.test.ts

```diff
@@ -188,4 +230,27 @@ describe('amplify add api (GraphQL)', () => {
       expect(err.message).toBe(`GraphQL API ${GraphQLAPIIdOutput} not found.`);
     }
   });
+
+  it('creates AppSync API in headless mode with non-capitalized model', async () => {
```

Comment:
Can we add this test to one of the existing schema? Spinning 3 new projects for this test increases the resource usage.